boards/xtensa/esp32s3_devkitm/esp32s3_devkitm.dts

```diff
@@ -5,7 +5,7 @@
  */
 /dts-v1/;
 
-#include <espressif/esp32s3/esp32s3_wroom_n8r8.dtsi>
+#include <espressif/esp32s3/esp32s3_mini_n8.dtsi>
```

Comment:
Is it a fixup of previous commit? Which SiP is the right one?

Reply:
Thanks, fixed!